tests/testthat/test-half.life.R

```diff
@@ -225,6 +225,25 @@ test_that("two-point half-life succeeds (fix #114)", {
   )
 })
 
+test_that("When tlast is excluded, lambda.z.time.last != tlast", {
+  d_conc <- data.frame(
+    conc = c(1, 0.5, 0.25, 0.1251, 0.05, 0),
+    time = c(0, 1, 2, 3, 4, 5),
+    exclude_hl = c(FALSE, FALSE, FALSE, FALSE, TRUE, FALSE),
+    subject = 1
+  )
+  d_dose <- data.frame(dose = 1, time = 0, subject = 1)
+
+  PKNCAconc <- PKNCAconc(d_conc, formula = conc ~ time | subject, exclude_half.life = "exclude_hl")
+  PKNCAdose <- PKNCAdose(d_dose, formula = dose ~ time | subject)
+  PKNCAdata <- PKNCAdata(PKNCAconc, PKNCAdose)
+  PKNCAresults <- pk.nca(PKNCAdata)

```

Comment:
For the assignments, please don't use the function names as that can easily break other tests. For example, please change

```
PKNCAconc <- PKNCAconc(d_conc, formula = conc ~ time | subject, exclude_half.life = "exclude_hl")
```

to

```
o_conc <- PKNCAconc(d_conc, formula = conc ~ time | subject, exclude_half.life = "exclude_hl")
```

(`o_conc` means object for concentration in the way that I think about it.)

Please do the same for PKNCAdose, PKNCAdata, and PKNCAresults.